medusa/search/proper.py

```diff
@@ -66,6 +67,8 @@ def run(self, force=False):  # pylint: disable=unused-argument
         if force:
```

Comment:
Using nzb doesn't influence that, but I agree with you propers download should and are something automated